.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
       - id: check-docstring-first
       - id: check-toml
       - id: check-yaml
-        exclude: packaging/.*
+        exclude: (packaging/.*|test/torchaudio_unittest/prototype/hifi_gan/original/.*)
```

Comment:
Can we format the reference implementation and get rid of this?
It's better to have less special treatment from the perspective of library maintenance cost.

Reply:
Formatted the ported code and removed the formatting exceptions here and in the code